tests/test_transaction.py

```diff
@@ -172,6 +172,49 @@ def test_no_combine_remaining(self):
         assert outputs[1][0] == RETURN_ADDRESS
         assert outputs[1][1] == 1000
 
+    def test_no_combine_remaining_small_inputs(self):
+        unspents_original = [Unspent(1500, 0, '', '', 0),
+                             Unspent(1600, 0, '', '', 0),
+                             Unspent(1700, 0, '', '', 0)]
+        outputs_original = [(RETURN_ADDRESS, 2000, 'satoshi')]
+
+        unspents, outputs = sanitize_tx_data(
+            unspents_original, outputs_original, fee=0, leftover=RETURN_ADDRESS,
+            combine=False, message=None
+        )
+        print(unspents)
```

Comment:
We can remove this now :slightly_smiling_face:

Reply:
Whoops! :)